man/rmd-fragments/authors-configuration.Rmd

```diff
@@ -0,0 +1,95 @@
+## Authors
+
+By default, pkgdown will display author information in three places:
+
+* the sidebar,
+* the left part side of the footer,
+* the author page.
+
+Both the sidebar and footer are customizable so you could move the authors 
+information or remove it entirely from them, see the sidebar and footer docs
+in respectively `?build_home` and `?build_footer`
+
+### Authors ORCID and bio
+
+Author ORCID identification numbers in the `DESCRIPTION` are linked using the ORCID logo:
+
+```
+Authors@R: c(
+    person("Hadley", "Wickham", , "hadley@rstudio.com", role = c("aut", "cre"),
+      comment = c(ORCID = "0000-0003-4757-117X")
+    ),
+    person("Jay", "Hesselberth", role = "aut",
+      comment = c(ORCID = "0000-0002-6299-179X")
+    )
+  )
+```
+
+If you want to add more details about authors or their involvement with the package,
+you can use the comment field,
+that will be rendered on the authors page.
+
+```
+Authors@R: c(
+    person("Hadley", "Wickham", , "hadley@rstudio.com", role = c("aut", "cre"),
+      comment = c(ORCID = "0000-0003-4757-117X", "Indenter-in-chief")
+    ),
+    person("Jay", "Hesselberth", role = "aut",
+      comment = c(ORCID = "0000-0002-6299-179X")
+    )
+  )
+```
+
+### YAML config - authors
+
+You can tweak a few things via the `authors` YAML field:
+
+* display of authors in the footer, sidebar and authors page,
```

Comment:
Which authors are displayed? Or how each author is display?

Reply:
rephrased